src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Rules/PropertyPages/PackagePropertyPage.xaml

```diff
@@ -27,6 +27,13 @@
                 Description="Produce a package file during build operations."
                 Category="General" />
 
+  <!-- TODO add a fwlink for this help URL -->
+  <BoolProperty Name="IsPublishable"
+                DisplayName="Allow project publishing (Publish target)"
+                Description="When publishing outside of Visual Studio, this allows commands such as &quot;dotnet publish&quot; to execute on this project."
+                HelpUrl="https://docs.microsoft.com/en-us/visualstudio/msbuild/msbuild-targets"
```

Comment:
This URL is not very helpful. If there are no docs for this, I'd remove this URL and file an issue on the docs repo to add a description for this property, tagging the relevant folks to chime in with a correct description of it.

Reply:
Created an issue. Will link it in a comment for this member. https://github.com/MicrosoftDocs/visualstudio-docs/issues/6643

Reply:
Done.